src/admin-views/editor/panel/ticket.php

```diff
@@ -121,9 +177,22 @@ class="ticket_form_title tribe-dependent"
 						name='ticket_name'
 						class="ticket_field ticket_form_right"
 						size='25'
-						value="<?php echo esc_attr( $ticket ? $ticket->name : null ); ?>"
+						value="<?php echo esc_attr( $ticket_name ); ?>"
 						data-validation-is-required
-						data-validation-error="<?php echo esc_attr( sprintf( _x( '%s type is a required field', 'admin edit ticket panel error', 'event-tickets' ), tribe_get_rsvp_label_singular( 'admin_edit_ticket_panel_error' ) ) ); ?>"
+						data-validation-error="
+						<?php
+						echo esc_attr(
+							sprintf(
+								// Translators: %s: dynamic 'RSVP' text.
+								_x(
+									'%s type is a required field',
+									'admin edit ticket panel error',
+									'event-tickets'
+								),
+								tribe_get_rsvp_label_singular( 'admin_edit_ticket_panel_error' )
+							)
+						);
+						?>"
```

Comment:
:no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 7be9cb2b7bf02fd7092d424e0b82aabc</sub>